src/allotropy/allotrope/schema_mappers/adm/cell_counting/benchling/_2023/_11/cell_counting.py

```diff
@@ -0,0 +1,200 @@
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from allotropy.allotrope.models.adm.cell_counting.benchling._2023._11.cell_counting import (
+    CellCountingAggregateDocument,
+    CellCountingDetectorDeviceControlAggregateDocument,
+    CellCountingDetectorMeasurementDocumentItem,
+    CellCountingDocumentItem,
+    DataProcessingDocument,
+    DataSystemDocument,
+    DeviceControlDocumentItemModel,
+    DeviceSystemDocument,
+    MeasurementAggregateDocument,
+    Model,
+    ProcessedDataAggregateDocument1,
+    ProcessedDataDocumentItem,
+    SampleDocument,
+)
+from allotropy.allotrope.models.shared.definitions.custom import (
+    TQuantityValueCell,
+    TQuantityValueMicrometer,
+    TQuantityValueMillionCellsPerMilliliter,
+    TQuantityValuePercent,
+    TQuantityValueUnitless,
+)
+from allotropy.allotrope.models.shared.definitions.definitions import TDateTimeValue
+from allotropy.constants import ASM_CONVERTER_VERSION
+from allotropy.parsers.utils.values import quantity_or_none
+
+
+@dataclass
+class Measurement:
+    measurement_identifier: str
+    timestamp: str
+    sample_identifier: str
+    analyst: str
+    viability: float
+    viable_cell_density: float
+    cell_type_processing_method: str | None = None
+    minimum_cell_diameter_setting: float | None = None
+    maximum_cell_diameter_setting: float | None = None
+    cell_density_dilution_factor: float | None = None
+    total_cell_count: float | None = None
+    total_cell_density: float | None = None
+    average_total_cell_diameter: float | None = None
+    average_live_cell_diameter: float | None = None
+    viable_cell_count: float | None = None
+    average_total_cell_circularity: float | None = None
+    average_viable_cell_circularity: float | None = None
+
+
+@dataclass(frozen=True)
+class MeasurementGroup:
+    measurements: list[Measurement]
+    _analyst: str | None = None
+
+    @property
+    def analyst(self) -> str | None:
+        if self._analyst is not None:
+            return self._analyst
+        if (
+            self.measurements
+            and len({m.analyst for m in self.measurements}) == 1
```

Comment:
Yup. I should (and will) add a comment here. The idea here is that if there is a single measurement time for all docs, then put it in the aggregate document, but otherwise don't.